include/swift/Sema/ConstraintSystem.h

```diff
@@ -3671,6 +3671,19 @@ class ConstraintSystem {
   bool isActiveTypeVariable(TypeVariableType *typeVar) const {
     return TypeVariables.count(typeVar) > 0;
   }
+  
+  /// New function to perform namelookup using the ResultBuilderContext
+  /// for the lookup. Binding will be defered to CSApply.
+  ///
+  /// This differs from the original function where we bind an UnresolvedDeclRefExpr by performing name lookup,
+  /// returning the resultant expression and using  context was the DeclContext used
+  /// for the lookup.
+  ///
+  /// \param replaceInvalidRefsWithErrors Indicates whether it's allowed
+  /// to replace any discovered invalid member references with `ErrorExpr`.
+  static Expr *resolveDeclRefExprNew(Expr *expr, UnresolvedDeclRefExpr *UDRE,
```

Comment:
Did you mean to commit this in this PR? 🙂

Reply:
😆 oh no